exec/java-exec/src/main/java/org/apache/drill/exec/rpc/data/DataConnectionCreator.java

```diff
@@ -66,8 +67,8 @@ public DataTunnel getTunnel(DrillbitEndpoint endpoint) {
     return new DataTunnel(newManager);
   }
 
-  public void close() {
-    Closeables.closeQuietly(server);
+  public void close() throws IOException {
```

Comment:
Yes, ```Closeables.close(..., true)``` does indeed swallow any IOException. However, the ```Closeables.close()``` method still declares that it throws IOException and so we have to deal with that somehow. 

So what would you recommend? We can either add a redundant catch block for IOException that we know will never be executed, or we can modify the ```DataConnectionManager.close()``` to declare it to throw IOE.